dataikuapi/dss/admin.py

```diff
@@ -1169,6 +1171,38 @@ def delete(self):
         return self.client._perform_empty(
             "DELETE", "/personal-api-keys/%s" % self.id_)
 
+class DSSPersonalApiKeyListItem(DSSTaggableObjectListItem):
```

Comment:
An API key is not a `taggable` object. It does not have tags. 
FYI: `com.dataiku.dip.server.services.TaggableObjectsService.TaggableObject`
IMO: could simply extends a `dict`.
Should be